examples/displaySegmentation/index.html

```diff
@@ -62,8 +62,7 @@ <h1>Display DICOM Segmentation IOD</h1>
         <script src="https://unpkg.com/hammerjs/hammer.min.js"></script>
 
         <script src="../js/initCornerstone.js"></script>
-
-        <script src="../js/dcmjs.js"></script>
+        <script src="../../build/dcmjs.js"></script>
```

Comment:
When trying the netlify preview for this PR https://deploy-preview-187--dcmjs.netlify.app/displaysegmentation/ I get the error 
```
Uncaught ReferenceError: dcmjs is not defined
    at parseSeg ((index):128)
    at XMLHttpRequest.<anonymous> ((index):97)
parseSeg @ (index):128
(anonymous) @ (index):97
load (async)
getAndLoadSeg @ (index):96
onclick @ (index):25
```
Not sure if having it work on the netlify preview is important, but I think that `<script src="../js/dcmjs.js"></script>` might fix it given the behavior in https://deploy-preview-185--dcmjs.netlify.app/displaysegmentation/

Reply:
Looking at the build rules in package.json, it looks like when you run npm run build:examples it expands to
`npm run build && npx cpx 'build/**/*.{js,map}' examples/js`
in which case I believe we should be fine to import from ../js/* in the index.html files

other than that, tested it locally and it works great for the displaySegmentation example!

Reply:
oh yeah, you are right, I was testing locally. thanks. I will switch back to src="../js/dcmjs.js"